pkg/config/redirect_config.go

```diff
@@ -70,13 +63,9 @@ func (c *RedirectConfig) ObservabilityExporterConfig() *observability.Config {
 }
 
 func (c *RedirectConfig) HostnameToRegion() (map[string]string, error) {
```

Comment:
Document that this is lowercase? Or document that `HostnameConfig` is case-insensitive.